DeepMRSeg/deepmrseg_apply.py

```diff
@@ -0,0 +1,136 @@
+#!/usr/bin/python3
+
+# deepmrseg
+
+import argparse as _argparse
+import os as _os
+import sys as _sys
+import urllib.request
+from urllib.parse import urlparse
+import zipfile
+from . import deepmrseg_test
+
+##############################################################
+## Path to saved models
+## FIXME: It would be better toread this from an external table hosted in model repo
+
+DEEPMRSEG = _os.path.expanduser(_os.path.join('~', '.deepmrseg'))
+MDL_DIR = _os.path.join(DEEPMRSEG, 'trained_models')
+
+# Temp for testing
+# MDL_DIR = '/home/guray/Documents/SBIA/Dev/DeepMRSeg/MyModels/ModelRepo'
```

Comment:
Any hard-coded paths should be removed. Especially ones from your local machine

Reply:
Fixed